### ISSUE TITLE
Crash in AttachmentViewHolder

### ISSUE THREAD

**Issue by spvince** [GetStream/stream-chat-android#507]
<h2>What did you do?</h2>
I open chrome and i copy url www.google.com. 
Then I paste the link in a channel.

<h2>What did you expect to happen?</h2>
No crash

<h2>What happened instead?</h2>
I have a crash in the code below because getType return null

<br/> <details>
<summary>Class: AttachmentViewHolder - Method : configAttachment() </summary>
<p>

```java
        boolean hasFile = false;
        boolean hasMedia = false;
        for (Attachment attachment : message.getAttachments()) {
            if (attachment.getType().equals(ModelType.attach_unknown)) continue;
            if (attachment.getType().equals(ModelType.attach_file)) {
                hasFile = true;
            } else {
                hasMedia = true;
            }
        }
```
</p>
</details>

![Capture d’écran 2020-05-28 à 17 06 00](https://user-images.githubusercontent.com/63065444/83161634-f88caa80-a108-11ea-9c49-4e70840ed9f0.png)

<h2>GetStream Environment</h2>

GetStream Chat version:
4.0.14

**Comment by elevenetc** [comment]
Thanks for report. Will check it.

**Comment by samiuelson** [comment]
@spvince Fix was merged and the new version of library published: `4.2.9-beta-3`. 

Please test and let us know if you have any further issues.

**Comment by samiuelson** [comment]
We are also working on the new version of the sample app, along with revamping UI components and view models. It should be ready for testing soon.
